crowbar_framework/lib/crowbar/lock.rb

```diff
@@ -16,33 +16,14 @@
 #
 
 module Crowbar
-  class Lock
-    attr_accessor :logger, :path, :name
-    attr_reader :path, :name, :file
+  module Lock
+    autoload :Base,
```

Comment:
No objection, but for my own understanding I'd like to know why this `autoload` is necessary.  If the answer is "go and read that long web page on Rails autoloading" then that's fine, although a TL;DR version would be really nice and helpful if possible ;-)  Thanks.